SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/TestActors/TestMovementCharacter.cpp

```diff
@@ -25,8 +28,8 @@ ATestMovementCharacter::ATestMovementCharacter()
 	SphereComponent->SetVisibility(true);
 	SphereComponent->SetupAttachment(GetCapsuleComponent());
 
-	FVector CameraLocation = FVector(300.0f, 0.0f, 75.0f);
-	FRotator CameraRotation = FRotator::MakeFromEuler(FVector(0.0f, -10.0f, 180.0f));
+	FVector CameraLocation = FVector(300.0f, 300.0f, 75.0f);
+	FRotator CameraRotation = FRotator::MakeFromEuler(FVector(0.0f, -10.0f, 225.0f));
```

Comment:
did we have it wrong?

Reply:
I think the new spawning just rotated the actors slightly... I just adjusted the camera accordingly.

Reply:
Ah, okay, just that this is used by other tests as well (unfortunately). So not sure what the effects elsewhere are.
It's a sphere anyway, so kind of hard to judge.